.devcontainer/devcontainer.json

```diff
@@ -1,26 +1,31 @@
 // For format details, see https://aka.ms/devcontainer.json. For config options, see the
 // README at: https://github.com/devcontainers/templates/tree/main/src/docker-existing-dockerfile
 {
-	"name": "Existing Dockerfile",
```

Comment:
Not sure whose formatter is correct here for .json files

Reply:
we should add a some kindof config file for JSON tied to a preferred formatter to avoid this maybe

Reply:
Ya. I'll make a separate PR to enforce the use of prettier on the repo. We already use it for all the JS projects.